components/amqpinflux/scripts/create.py

```diff
@@ -67,7 +69,7 @@ def install_amqpinflux():
     ctx.logger.info('Fixing permissions...')
     utils.chown(amqpinflux_user, amqpinflux_group, AMQPINFLUX_HOME)
 
-    utils.systemd.configure('amqpinflux')
```

Comment:
Just to add on top of that for clarity's sake. What I meant is that the parameters can be generated dynamically according to which service you're currently on. This can happen in `download_resource_and_render` and anywhere else where the params are used.